homeassistant/components/emulated_roku/.translations/en.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
These are autogenerated files and will be overridden, only `strings.json` matters.